### DISCUSSION TITLE
WIZ850io, Works fine with Ethernet.h, but not with Ethernet3.h on ESP32

### DISCUSSION THREAD

**Discussion by zamorae** [sstaub/Ethernet3#46] [Q&A]
Hi, I have purchased a couple of WIZ850io boards that work perfect using Paul Stoffregen's Ethernet.h (@ version 2.0.0), but when I try the Ethernet3.h (@current version 1.5.5)  using the Ethernet3-master library it simply stops working. I am using Arduino IDE 1,8,10 and using GPIO 5 as CS. The program example I use to run the test is UDPSendReceive.ino (in the examples folder on this same repository from sstaub), which is very similar to the one in PS Ethernet's  github folder (see link below), with the Exception of the setup line Ethernet.init(5).
 Is there a specific pin arrangement I need to observe to connect the WIZ850io to an ESP32 to use Ethernet3? 
I am using a DevKit. (ESP32-WROOM-32U) The Wiz850 runs fine using this code:
[https://github.com/PaulStoffregen/Ethernet/blob/master/examples/UDPSendReceiveString/UDPSendReceiveString.ino](url)
My SPI pinout a ESP32 is as follows:
**_CS -> 5
RST -> 12
CLK -> 18
MISO -> 19
MOSI -> 23_**

I use the sketch mentioned above to check udp messages sent between my android phone and the ESP32. Using same ports (8888) and the same IP address on the programs (192.168.1.177). As I mentioned, works great with Ethernet.h but I'd like to get it working with Ethernet3.h
Thanks in advance
-Ed

**Comment by sstaub** [comment]
maybe Ethernet.init(5) is the problem, try Ethernet.init(4)
